dlc/src/test/scala/org/bitcoins/dlc/BinaryOutcomeDLCWithSelfIntegrationTest.scala

```diff
@@ -189,7 +191,8 @@ class BinaryOutcomeDLCWithSelfIntegrationTest extends BitcoindRpcTest {
         localLosePayout = localInput - CurrencyUnits.oneMBTC,
         timeout = tomorrowInBlocks,
         feeRate = feeRate,
-        changeSPK = changeSPK,
+        localChangeSPK = localChangeSPK,
+        remoteChangeSPK = remoteChangeSPK,
         network = RegTest
```

Comment:
We probably should have assertions in this test case that the resulting signed tx does not contain the empty spk. Same with `BinaryOutcomeDLCWithSelfTest`